lib/sqlite.core.js

```diff
@@ -501,6 +501,16 @@ SQLitePluginTransaction.prototype.addStatement = function(sql, values, success,
       } else if (t === 'boolean') {
         //Convert true -> 1 / false -> 0
         params.push(~~v);
+      } else if (Array.isArray(v)) {
+        s = '(';
+        for (i = 0, len2 = v.length; len2 > 1 && i < len2 - 1; i++) {
+          s = s + v[i].toString() + ',';
+        }
+        if (len2 > 0) {
+          s = s + v[len2 - 1];
+        }
+        s = s + ')';
+        params.push(s);
```

Comment:
so a couple of things. 

1. if the array is empty, we will generate "()" - is this an expected behavior ?
2. Doing toString() inside the conversion loop essentially coerces all values into strings - is this what you had in mind? I understand that the values in the array should be of the same type, but what if they are numbers or boolean ? Should we handle these cases as well ?